app/assets/javascripts/react/components/CirculationKiosk.js

```diff
@@ -0,0 +1,24 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import Kiosk from './presentational/Circulation/Kiosk';
+import * as actions from '../actions';
+
+// Top level mapping of the application state to properties of component that is being
+// connected.
+const mapStateToProps = (state) => {
+  return {
+    slides: state.kiosk.slides,
+    url: state.kiosk.url,
+    is_fetching_slides: state.touch.is_fetching_slides,
+    show_nav: false
```

Comment:
You want to set the default values for state the relate reducer, and then pull the current value here as `state.circulation.show_nav`. You would then create a `reducers/circulationReducer.js` similar to the other reducers.. and add the new circulationReducer to `reducers/index.js` so that it is bound what redux provides as a rootReducer.